e2e/testsuite/testsuite.go

```diff
@@ -229,6 +229,12 @@ func (s *E2ETestSuite) StartRelayer(relayer ibc.Relayer) {
 	s.startRelayerFn(relayer)
 }
 
+// StopRelayer stops the given relayer.
+func (s *E2ETestSuite) StopRelayer(ctx context.Context, relayer ibc.Relayer, eRep ibc.RelayerExecReporter) {
+	err := relayer.StopRelayer(ctx, eRep)
```

Comment:
if we're creating a separate function then we can probably remove `eRep` as an argument

```golang
func (s *E2ETestSuite) StopRelayer(ctx context.Context, relayer ibc.Relayer) {
	err := relayer.StopRelayer(ctx, s.GetRelayerExecReporter())
        s.Require().NoError(err)
}
```

Reply:
Nice catch! I'll push an update. 

I was thinking it might be an idea to eventually move the ctx onto the `E2ETestSuite` as well, as I think it's just being retrieved from `context.TODO()` in most test cases.

Reply:
yeah atm it's just doing a passthrough to the ibctest functions, but I think having the context at the test level can be useful. It allows for things like timeouts and cancellations. (we just aren't using it for anything atm)